packages/newspringchurchapp/src/prayer/UserPrayerList/index.js

```diff
@@ -43,18 +45,62 @@ class UserPrayerList extends React.Component {
                     <H6>Viewing</H6>
                     <GreenH4>My Prayers</GreenH4>
                   </PaddedView>
-                  <StyledView>
-                    {userPrayers
-                      .map((prayer) => (
-                        <Card key={prayer.id}>
-                          <CardContent>
-                            {/* TODO add action prop containing deletePrayer button */}
-                            <PrayerSingle prayer={prayer} showDate />
-                          </CardContent>
-                        </Card>
-                      ))
-                      .reverse()}
-                  </StyledView>
+                  <Mutation
+                    mutation={DELETE_PRAYER}
+                    update={async (cache, { data: { deletePrayer } }) => {
```

Comment:
for some reason I don't think this needs to be `async/await` since the last item is already returning a promise

Reply:
Removed.